client/layout/guided-tours/tours/editor-insert-menu-tour.js

```diff
@@ -49,15 +51,13 @@ export const EditorInsertMenuTour = makeTour(
 			} }
 		>
 			<p>
-				{ translate( '{{strong}}Add Media{{/strong}} has moved to a new button.', {
-					components: { strong: <strong /> },
-					comment: 'Title of the Guided Tour for the Editor Insert Menu button.',
+				{ translate( '{{addMediaButton/}} has moved to a new button.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '{{AddMediaButton/}} has moved to a new button.' )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).